UT4MasterServer/Services/AccountService.cs

```diff
@@ -46,6 +46,12 @@ public async Task CreateAccountAsync(string username, string email, string passw
 		return await cursor.SingleOrDefaultAsync();
 	}
 
+	public async Task<IEnumerable<Account>> SearchAccountsAsync(string query)
+	{
+		var cursor = await accountCollection.FindAsync(account => account.Username.ToLower().Contains(query.ToLower()));

```

Comment:
Did you made sure searching like this works? i would think it doesn't since you do "so many" operations on the strings.

Reply:
Yep it works. Mongo supports toLower, but it doesn't support Contains(str, StringComparison.ignoreCase)